docker/dev/build.sh

```diff
@@ -1,14 +1,42 @@
 #!/bin/bash
+# Before run of this script you can set environmental variables
+# IMAGE_TAG, DOCKERFILE, BASE_IMG, GOLANG_OS_ARCH, .. then  export them
+# and to use defined values instead of default ones
 
 cd "$(dirname "$0")"
 
 set -e
 
-IMAGE_TAG=${IMAGE_TAG:-'dev_vpp_agent'}
+#IMAGE_TAG=${IMAGE_TAG:-'dev_vpp_agent'}
```

Comment:
Please keep the image-tag name, otherwise this is gonna break build for production image, the name is used as base image there. Let's keep this simple and just use `dev_vpp_agent`, it's local name anyway.

Reply:
ok